src/app/report/edit-display/edit-display.component.ts

```diff
@@ -36,25 +39,32 @@ import { NgIf, NgFor } from '@angular/common';
     CustomEditorComponent,
     DisplayTableComponent,
     NgFor,
+    BooleanToStringPipe,
+    NgStyle,
+    ClipboardModule,
   ],
 })
 export class EditDisplayComponent {
   editingEnabled: boolean = false;
   editingChildNode: boolean = false;
   editingRootNode: boolean = false;
+  metadataTableVisible: boolean = false;
   rerunResult: string = '';
   report: any = {};
+  displayReport: boolean = false;
   @Input() id: string = '';
-  currentView: any = {
-    storageName: 'Test',
-  };
-  @Output() saveReportEvent = new EventEmitter<any>();
+  @Input() containerHeight!: number;
+  @Input() currentView: any = {};
```

Comment:
Is there no type for this yet?

Reply:
Because making everything typed is solved in another PR, I did not make any chances types